sddi/Dockerfile

```diff
@@ -1,107 +1,193 @@
-# #############################################################################
-# # Build stage
-# #############################################################################
+###############################################################################
+# Extbuild stage
+###############################################################################
 ARG BASEIMAGE_REPOSITORY=ghcr.io/tum-gis/ckan-sddi-base
 ARG BASEIMAGE_VERSION=latest
-ARG CKAN_VERSION_BUILD_STAGE=2.9.9-dev
+ARG CKAN_VERSION_BUILD_STAGE=2.11.1
 
 FROM ckan/ckan-base:${CKAN_VERSION_BUILD_STAGE} as extbuild
 
 USER root
 
-# ckanext-geoview #############################################################
-ARG CKANEXT_GEOVIEW_VERSION="v0.0.20"
-ENV CKANEXT_GEOVIEW_VERSION=${CKANEXT_GEOVIEW_VERSION}
-ENV CKANEXT_GEOVIEW_GITHUB_URL="https://github.com/ckan/ckanext-geoview"
+RUN apt-get update && apt-get install -y \
+    curl \
+    libpq-dev \
+    autoconf \
+    automake \
+    libtool \
+    patch \
+    musl-dev \
+    libpcre3-dev \
+    libpcre3 \
+    libffi-dev \
+    libxml2-dev \
+    libxslt-dev
+
+RUN pip install -U markupsafe==2.0.1 sqlalchemy==1.4.41
```

Comment:
Official CKAN 2.11.1 uses different version for those packages. Is there a specific reason why you install those version?
If no, please remove this line.